api/auth/external_auth.proto

```diff
@@ -65,11 +65,13 @@ message AttributeContext {
 
   // Represents a network request, such as an HTTP request.
   message Request {
-    oneof request {
-      HTTPRequest http_request = 1;
-    }
     // The timestamp when the proxy receives the first byte of the request.
```

Comment:
The node that receives the request may not be a proxy. I suggest we use this comment.

// The timestamp when the first byte of the request is received.